src/variety/cbblock.js

```diff
@@ -24,10 +24,46 @@ MouseEvent:{
 		}
 	}
 },
+"MouseEvent@dbchoco":{
+	inputModes : {edit:['ice','number','clear'],play:['border','subline']},
```

Comment:
There's an issue with reusing `'ice'` input mode, in that the pzpr-ui calls the mode "Icebarn". I think the best solution would probably be to rework (and rename) the mode to something more generic. (Not to be fixed here, at least not at this point, more as a note to self.)

Reply:
I see now that herugolf uses its own `'water'` input mode, which is then labelled "Water hazard". I don't think we want to add a dedicated input mode per background shade though.

Reply:
Ok I think what I'd like to do here is switch the input mode over to 'shade', and generalize the that input mode to either work directly for this type, or to allow us to override some `inputShade()` method here. But that doesn't need to happen before merging, I'm happy to do that some time later.